src/game/client/components/nameplates.cpp

```diff
@@ -177,7 +177,11 @@ void CNamePlates::OnRender()
 
 		if(m_pClient->m_aClients[i].m_SpecCharPresent)
 		{
-			RenderNameplatePos(m_pClient->m_aClients[i].m_SpecChar, pInfo, 0.4f);
+			bool OtherTeam = m_pClient->IsOtherTeam(i);
+			float Alpha = OtherTeam ? g_Config.m_ClShowOthersAlpha / 100.0f : 0.4f;
```

Comment:
```suggestion
			float Alpha = 0.4f * (OtherTeam ? g_Config.m_ClShowOthersAlpha / 100.0f : 1.0f);
```

Would be more in line with the tee rendering.